bootstrap/ansible/playbook.yml

```diff
@@ -216,6 +216,18 @@
           become_user: "{{ djangooperator }}"
           changed_when: venv_register.stdout == "created"
 
+        # Install psycopg2 and mod-wsgi, which are required to create the
```

Comment:
I'd prefer not to have the specific modules and versions duplicated here. Given that this is intended to run once, can we install all Pip requirements here?

We'd still need to install them in common tasks, as requirements may change.